src/controllers/user.js

```diff
@@ -13,9 +16,8 @@ const signUp = async (req, res) => {
             familyName: user.familyName,
             dateOfBirth: user.dateOfBirth,
             address: user.address,
-            latitude: user.latitude,
-            longitude: user.longitude,
         },
+        addToCoordinate,
```

Comment:
Is it possible to put addToCoordinate inside the user object?

Reply:
I am not sure if it is required or possible! lets have Jacob idea on this.

Reply:
@jpremo is it possible to return addToCoordinate inside the user object? or is it supposed to return separately like this? Could you please advise? Thank you.